src/allotropy/parsers/thermo_skanit/thermo_skanit_structure.py

```diff
@@ -0,0 +1,277 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+import re
+
+import pandas as pd
+
+from allotropy.allotrope.models.shared.components.plate_reader import SampleRoleType
+from allotropy.allotrope.schema_mappers.adm.plate_reader.benchling._2023._09.plate_reader import (
+    Data,
+    Measurement,
+    MeasurementGroup,
+    MeasurementType,
+    Metadata,
+)
+from allotropy.exceptions import AllotropyParserError
+from allotropy.parsers.utils.uuids import random_uuid_str
+
+sample_role_mappings = {
+    "Un": SampleRoleType.unknown_sample_role,
+    "Std": SampleRoleType.standard_sample_role,
+    "Blank": SampleRoleType.blank_role,
+    "Ctrl": SampleRoleType.control_sample_role,
+}
+
+
+def find_value_by_label_optional(df: pd.DataFrame, label: str) -> str | None:
+    row = df[df.iloc[:, 1].str.contains(label, case=False, na=False)]
+    if not row.empty:
+        return str(row.iloc[0, 4])  # Assuming the value is in the 5th column (index 4)
+    return None
+
+
+def find_value_by_label_required(df: pd.DataFrame, sheet_name: str, label: str) -> str:
+    row = df[df.iloc[:, 1].str.contains(label, case=False, na=False)]
+    if not row.empty:
+        return str(row.iloc[0, 4])  # Assuming the value is in the 5th column (index 4)
+    else:
+        msg = f"Unable to identify {label} from {sheet_name} tab"
+        raise AllotropyParserError(msg)
+
+
+@dataclass(frozen=True)
+class ThermoSkanItMetadata:
+    @staticmethod
+    def create_metadata(
+        instrument_info_df: pd.DataFrame, general_info_df: pd.DataFrame, file_name: str
+    ) -> Metadata:
+        software_info = None
```

Comment:
If we clean up the dataframes a bit (see below) we can parse these into `SeriesData` objects and access the data by the labels:

```
        # Replace empty with "" so we can add label columns together.
        instrument_info_df = instrument_info_df.fillna("")
        # The labels for data is spread across the first two columns for some reason, combine them as index.
        instrument_info_df.index = instrument_info_df.iloc[:, 0] + instrument_info_df.iloc[:, 1]
        # Read data from the last column
        instrument_info_data = df_to_series_data(instrument_info_df.T, index=-1)

        general_info_data = df_to_series_data(parse_header_row(general_info_df.T))
```